releases/nginx-ingress.yaml

```diff
@@ -209,8 +211,7 @@ releases:
         enabled: true
         labels:
           grafana_dashboard: "true"
-
-  set:
-  - name: "configMaps.dashboards.files.nginx\\.json"
-    file: https://raw.githubusercontent.com/cloudposse/grafana-dashboards/{{ env "GRAFANA_DASHBOARDS_VERSION" | default "1.0" }}/nginx-ingress/nginx.json
+        files:
+          nginx-ingress.json: |
+{{ exec "curl" (list "-sSL" (print "https://raw.githubusercontent.com/cloudposse/grafana-dashboards/" (env "GRAFANA_DASHBOARDS_VERSION" | default "1.0") "/nginx-ingress/nginx.json")) | indent 12 }}
```

Comment:
```suggestion
{{ exec "curl" (list "-fsSL" (print "https://raw.githubusercontent.com/cloudposse/grafana-dashboards/" (env "GRAFANA_DASHBOARDS_VERSION" | default "1.0") "/nginx-ingress/nginx.json")) | indent 12 }}
```

Reply:
Thoughts on using `printf` instead?

Reply:
`printf` is unnecessarily abstract. In my opinion
```
print "x" "y" "x"
```
is preferable for readability and mainatinability over
```
printf "x%sz" "y"
```

Reply:
Thought that's slightly unfair since in this case `/` is a quite nice visual delimiter.  So in the end, it looks something like:
```
(env "GRAFANA_DASHBOARDS_VERSION" | default "1.0") | printf "https://raw.githubusercontent.com/cloudposse/grafana-dashboards/%s/nginx-ingress/nginx.json" | indent 12 }}
```

But no strong feelings. Fine to leave as-is